openapi-mock/README.md

```diff
@@ -5,131 +5,30 @@
 # OpenAPI Mock
 [![npm version](https://badge.fury.io/js/varkes-openapi-mock.svg)](https://badge.fury.io/js/varkes-openapi-mock)
 
-Mocks res calls given an OpenAPI specification 
+The OpenAPI mock mocks the application APIs based on the OpenAPI specification.
 
-After startup, you have access to:
-- rest API - http://localhost:10000/myApiBasePath/metadata
-- console - http://localhost:10000/myApiBasePath/console
+## Features
 
-## Starting the application
+The OpenAPI mock brings you the following features:
 
-### Run local
+- OpenAPI Mock uses [Swagger-Express-Middleware](https://github.com/BigstickCarpet/swagger-express-middleware) to parse, validate, and dereference OpenAPI files.  You can also create your custom implementation for responses or errors.
 
-To run it local, specify a config file (as located in the /test folder) like that:
-```
-npm install
-node server/server <varkes_config.js>
-```
+- Records every request made to the node: The mock uses [morgan](https://www.npmjs.com/package/morgan)logging framework to create  a `requests.log` file. This file contains the called URLs, the request header, and the request body if exists.
 
-### Run local using docker
+- Returns the OpenAPI specification as metadata: By calling '/metadata' you can see the OpenAPI specification in `text` or `x-yaml` format.
 
-To run it using docker, call:
-```
-docker run -p 10000:10000 eu.gcr.io/kyma-project/incubator/develop/varkes-openapi-mock:latest
-```
+- Returns a dummy OAuth2 token: By calling the '/authorizationserver/oauth/token' endpoint and adding the OAuth2 requirements as query parameters you can get a dummy OAuth2 token.
 
-### Run in Kyma
 
-To run the mock using Kyma as runtime envrironment, run the following kubectl command to set up a namespace:
+## Run mock
 
-```bash
-kubectl create namespace mocks
-kubectl label namespace mocks env=true
-```
 
-and to deploy the mock
-```bash
-kubectl apply -n mocks -f https://raw.githubusercontent.com/kyma-incubator/varkes/master/openapi-mock/deployment/deployment.yaml
+To run the mock locally, specify the [varkes config](https://github.com/kyma-incubator/varkes/blob/master/openapi-mock/test/varkes_config.json) file:
```

Comment:
Replace with instructions provided to app-connector-client

Reply:
Done.